<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AircraftPerfDialog</class>
 <widget class="QDialog" name="AircraftPerfDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Little Navmap - Aircraft Performance</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="labelName">
       <property name="text">
        <string>&amp;Name:</string>
       </property>
       <property name="buddy">
        <cstring>lineEditName</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditName"/>
     </item>
     <item>
      <widget class="QLabel" name="labelType">
       <property name="text">
        <string>Aircraft &amp;Type:</string>
       </property>
       <property name="buddy">
        <cstring>lineEditType</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditType">
       <property name="toolTip">
        <string>Aircraft type like B732 or C172</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Fuel</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>2</number>
      </property>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxFuelUnit">
        <item>
         <property name="text">
          <string>Weight (%weight%)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Volume (%volume%)</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelFuelUnits">
        <property name="text">
         <string>&amp;Fuel Units:</string>
        </property>
        <property name="buddy">
         <cstring>comboBoxFuelUnit</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="spinBoxExtraFuel">
        <property name="toolTip">
         <string>Fuel added at the discretion of the
captain or the dispatcher</string>
        </property>
        <property name="suffix">
         <string> %fuel%</string>
        </property>
        <property name="maximum">
         <number>1000000</number>
        </property>
        <property name="singleStep">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="labelCruiseExtraFuel">
        <property name="text">
         <string>&amp;Extra fuel:</string>
        </property>
        <property name="buddy">
         <cstring>spinBoxExtraFuel</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelReserveFuel">
        <property name="text">
         <string>&amp;Reserve Fuel:</string>
        </property>
        <property name="buddy">
         <cstring>spinBoxReserveFuel</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelTaxiFuel">
        <property name="text">
         <string>&amp;Taxi fuel:</string>
        </property>
        <property name="buddy">
         <cstring>spinBoxTaxiFuel</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="spinBoxReserveFuel">
        <property name="toolTip">
         <string>Minimum fuel required to fly for 30 minutes at 1,500 feet
above the alternate or at the destination at holding speed if
an alternate is not required</string>
        </property>
        <property name="suffix">
         <string> %fuel%</string>
        </property>
        <property name="maximum">
         <number>1000000</number>
        </property>
        <property name="singleStep">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="spinBoxTaxiFuel">
        <property name="toolTip">
         <string>Fuel used prior to takeoff by APU, engine start and taxi</string>
        </property>
        <property name="suffix">
         <string> %fuel%</string>
        </property>
        <property name="maximum">
         <number>1000000</number>
        </property>
        <property name="singleStep">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QSpinBox" name="spinBoxContingencyFuel">
        <property name="toolTip">
         <string>This accounts for additional enroute fuel consumption
caused by wind or routing changes.
This will increase the trip fuel by the given percentage.</string>
        </property>
        <property name="suffix">
         <string> percent</string>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="singleStep">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="labelExtraCruiseFuel">
        <property name="text">
         <string>&amp;Contingency fuel:</string>
        </property>
        <property name="buddy">
         <cstring>spinBoxContingencyFuel</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBoxFuelType">
        <item>
         <property name="text">
          <string>Avgas</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Jetfuel</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelFuelType">
        <property name="text">
         <string>&amp;Fuel Type:</string>
        </property>
        <property name="buddy">
         <cstring>comboBoxFuelType</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Climb</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>2</number>
      </property>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinBoxClimbSpeed">
        <property name="suffix">
         <string> %speed% true airspeed</string>
        </property>
        <property name="minimum">
         <number>20</number>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelClimbSpeed">
        <property name="text">
         <string>Average &amp;climb speed:</string>
        </property>
        <property name="buddy">
         <cstring>spinBoxClimbSpeed</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelVertClimbSpeed">
        <property name="text">
         <string>Average vertical &amp;climb speed:</string>
        </property>
        <property name="buddy">
         <cstring>spinBoxClimbVertSpeed</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="spinBoxClimbVertSpeed">
        <property name="suffix">
         <string> %vspeed% ▲</string>
        </property>
        <property name="minimum">
         <number>100</number>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="singleStep">
         <number>100</number>
        </property>
        <property name="value">
         <number>550</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="labelClimbFuelFlow">
        <property name="text">
         <string>Average fuel flow for &amp;climb:</string>
        </property>
        <property name="buddy">
         <cstring>spinBoxClimbFuelFlow</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="spinBoxClimbFuelFlow">
        <property name="suffix">
         <string> %fuel% per hour</string>
        </property>
        <property name="maximum">
         <number>100000</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Cruise</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>2</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="labelCruiseSpeed">
        <property name="text">
         <string>C&amp;ruise speed at typical cruise altitude:</string>
        </property>
        <property name="buddy">
         <cstring>spinBoxCruiseSpeed</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinBoxCruiseSpeed">
        <property name="suffix">
         <string> %speed% true airspeed</string>
        </property>
        <property name="minimum">
         <number>20</number>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelCruiseFuelFlow">
        <property name="text">
         <string>Fuel flow in c&amp;ruise:</string>
        </property>
        <property name="buddy">
         <cstring>spinBoxCruiseFuelFlow</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinBoxCruiseFuelFlow">
        <property name="suffix">
         <string> %fuel% per hour</string>
        </property>
        <property name="maximum">
         <number>100000</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Descent</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>2</number>
      </property>
      <item row="3" column="0">
       <widget class="QLabel" name="labelDescentVertSpeed">
        <property name="text">
         <string>Average vertical &amp;descent speed:</string>
        </property>
        <property name="buddy">
         <cstring>spinBoxDescentVertSpeed</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="spinBoxDescentVertSpeed">
        <property name="suffix">
         <string> %vspeed% ▼</string>
        </property>
        <property name="prefix">
         <string/>
        </property>
        <property name="minimum">
         <number>-10000</number>
        </property>
        <property name="maximum">
         <number>-100</number>
        </property>
        <property name="singleStep">
         <number>100</number>
        </property>
        <property name="value">
         <number>-550</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelDescentSpeed">
        <property name="text">
         <string>Average &amp;descent speed:</string>
        </property>
        <property name="buddy">
         <cstring>spinBoxDescentSpeed</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinBoxDescentSpeed">
        <property name="suffix">
         <string> %speed% true airspeed</string>
        </property>
        <property name="minimum">
         <number>20</number>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="labelDescentRule">
        <property name="toolTip">
         <string>Equivalent rule of thumb for
the given descent speeds</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="labelDescentFuelFlow">
        <property name="text">
         <string>Average fuel flow for &amp;descent:</string>
        </property>
        <property name="buddy">
         <cstring>spinBoxDescentFuelFlow</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="spinBoxDescentFuelFlow">
        <property name="suffix">
         <string> %fuel% per hour</string>
        </property>
        <property name="maximum">
         <number>100000</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextBrowser" name="textBrowserDescription">
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="acceptRichText">
      <bool>false</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextEditorInteraction</set>
     </property>
     <property name="placeholderText">
      <string>Description or Comments</string>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok|QDialogButtonBox::Reset|QDialogButtonBox::RestoreDefaults</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditName</tabstop>
  <tabstop>lineEditType</tabstop>
  <tabstop>comboBoxFuelUnit</tabstop>
  <tabstop>comboBoxFuelType</tabstop>
  <tabstop>spinBoxReserveFuel</tabstop>
  <tabstop>spinBoxTaxiFuel</tabstop>
  <tabstop>spinBoxExtraFuel</tabstop>
  <tabstop>spinBoxContingencyFuel</tabstop>
  <tabstop>spinBoxClimbSpeed</tabstop>
  <tabstop>spinBoxClimbVertSpeed</tabstop>
  <tabstop>spinBoxClimbFuelFlow</tabstop>
  <tabstop>spinBoxCruiseSpeed</tabstop>
  <tabstop>spinBoxCruiseFuelFlow</tabstop>
  <tabstop>spinBoxDescentSpeed</tabstop>
  <tabstop>spinBoxDescentVertSpeed</tabstop>
  <tabstop>spinBoxDescentFuelFlow</tabstop>
  <tabstop>textBrowserDescription</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
